force-app/main/default/classes/UTIL_Describe.cls

```diff
@@ -576,41 +591,29 @@ public class UTIL_Describe {
                         new List<String>{'\''+fieldSetMember.getFieldPath()+'\'', '\''+fieldSetName+'\'', '\''+obj+'\''});
                     throw (new SchemaDescribeException(errorMessage));
                 }
-                if(!onlyRequired || fieldSetMember.getDBRequired() || fieldSetMember.getRequired()){
+                if(!onlyRequired || fieldSetMember.getDbRequired() || fieldSetMember.getRequired()){
                     fieldNames.add(fieldSetMember.getFieldPath());
                 }
             }
             return fieldNames;
         }
     }
 
-    /*******************************************************************************************************
-    * @description utility to return a comma separated string of field names from a field set
-    * @param strObject the name of the object
-    * @param strFS the name of the object's field set
-    * @return a comma separated string of field names in the field set
-    */
-    public static string strFieldNamesFromFieldSet(string strObject, string strFS) {
```

Comment:
Heh, no references to this, huh?

Reply:
Yep, I looked high and low for references when I saw it was greyed out in my IDE.